book/src/ncp.md

```diff
@@ -1,3 +1,71 @@
 # The Network Control Plane
 
-The Network Control Plane (NCP) implements a gossip network between all nodes on in the cluster.
+The Network Control Plane (NCP) is the gateway to the control plane. Any node
+may join the control plane. It is when nodes need to ensure information is
+available to *all* nodes in a Solana cluster. The control plane is implemented
```

Comment:
> ... is the gateway to the control plane. 

This isn't super clear. Network Control Plane is the gate way to the (???) control plane. We only have two types, Data and Network CP, no? Are you saying that the ncp is the gateway to the Data plane?
> It is when nodes need to ensure information to *all* nodes in a Solana cluster. 

[Suggestion] It is used _by nodes_ to ensure information is available ...
 
Maybe the whole first sentence could just be condensed to 
>The Network Control Plane (NCP) is used to ensure information is
available to *all* nodes in a Solana cluster.

Reply:
Like the TPU and TVU, the NCP is a component in fullnode. A more precise name for it would be Control Plane Gateway.

Reply:
Yeah it's more like a gateway to the control plane than the control plane itself.